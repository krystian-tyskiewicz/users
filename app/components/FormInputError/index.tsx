'use client'

import styled from 'styled-components'

const FormInputError = styled.span`
  position: absolute;
  left: 0;
  bottom: -2px;
  font-size: 12px;
  color: red;
`

export default FormInputError
