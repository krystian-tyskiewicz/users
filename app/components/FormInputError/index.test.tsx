import { render } from '@testing-library/react'
import FormInputError from '.'

it('renders correctly', () => {
  const { container } = render(<FormInputError>Test</FormInputError>)
  expect(container).toMatchSnapshot()
})
