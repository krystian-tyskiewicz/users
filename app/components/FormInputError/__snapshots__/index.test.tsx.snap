// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <span
    class="sc-bdvumH eDHdMN"
  >
    Test
  </span>
</div>
`;
